docs/readthedocs/source/doc/LLM/Overview/install_gpu.md

```diff
@@ -196,6 +196,40 @@ BigDL-LLM for GPU supports on Linux has been verified on:
                
                   sudo apt autoremove intel-basekit
 
+         .. tab:: PIP installer
+
+            Step 1: Install oneAPI in a user-defined folder, e.g., ``~/intel/oneapi``
+
+            .. code-block:: bash
+
+               export PYTHONUSERBASE=~/intel/oneapi
+               pip install dpcpp-cpp-rt==2024.0.2 mkl-dpcpp==2024.0.0 onednn==2024.0.0 --user
+
+            .. note::
+
+               The oneAPI packages are visible in ``pip list`` only if ``PYTHONUSERBASE`` is set.
+
+            Step 2: Configure conda environment activation to append ``~/intel/oneapi/lib`` to environment variable ``LD_LIBRARY_PATH``
+            In the example below, we first create conda environment ``llm`` and then configure it.
```

Comment:
Configure ,__your working__ conda environment (e.g. with name llm) __config__ to append ``~/intel/oneapi/lib`` to environment variable ``LD_LIBRARY_PATH``: